### PR TITLE
[updates] rm planifolia dep and update styling part 1

### PR DESCRIPTION
**Pull request by phillimorland**

### CODE REVIEW

**Review by fuzzylogic2000 — APPROVED** [review]
Code reads well and most things look the same.
Things I already niticed, but can be fixed later (maybe collect all in an issue?):
- "Save"-button in dashboard is not wide enough
- donation banner on LIQD (wagtail)-pages does not have background colour and disappers from time to time (depending on background of pages)
- blue of dashboard publish progress line seems to be wrong

**Comment by phillimorland** [comment]
> Code reads well and most things look the same. Things I already niticed, but can be fixed later (maybe collect all in an issue?):
> 
>     * "Save"-button in dashboard is not wide enough
> 
>     * donation banner on LIQD (wagtail)-pages does not have background colour and disappers from time to time (depending on background of pages)
> 
>     * blue of dashboard publish progress line seems to be wrong

I think last time I collected everything in the story but issue is same difference so whatever you think is best

**Comment by fuzzylogic2000** [comment]
> > Code reads well and most things look the same. Things I already niticed, but can be fixed later (maybe collect all in an issue?):
> > ```
> > * "Save"-button in dashboard is not wide enough
> > 
> > * donation banner on LIQD (wagtail)-pages does not have background colour and disappers from time to time (depending on background of pages)
> > 
> > * blue of dashboard publish progress line seems to be wrong
> > ```
> 
> I think last time I collected everything in the story but issue is same difference so whatever you think is best

I already added the issue. And it's maybe easier for testers to add things there?!

Nice changes on the linting, but now it's really the tests that are failing! :(

File: apps/documents/assets/__tests__/react_documents_render.jest.jsx [hunk 1]
```diff
@@ -1,21 +0,0 @@
-// checking for simple element to see if component is rendered as interim solution
-import '../../../../__mocks__/jsDomMock'
```
Comment (phillimorland) [inline apps/documents/assets/__tests__/react_documents_render.jest.jsx:2]: removed as hoping to redo all the tests and add the additional ones from mB in another PR, just not working yet https://github.com/liqd/adhocracy-plus/pull/2105

**Pull request merged**